src/mlagility/cli/slurm.py

```diff
@@ -74,7 +74,7 @@ def run_benchit(
     build_only: Optional[bool] = None,
     lean_cache: Optional[bool] = None,
     working_dir: str = os.getcwd(),
-    ml_cache_dir: Optional[str] = None,
+    ml_cache_dir: Optional[str] = os.environ.get("ML_CACHE"),
```

Comment:
```suggestion
    ml_cache_dir: Optional[str] = os.environ.get("MLAGILITY_MODEL_DOWNLOAD_DIR"),
```

Feel free to use a different name, but the general idea is to:
1. Namespace the env var so that we dont have collisions
2. Disambiguate from the `mlagility cache dir` which is something different entirely
Suggest avoiding namespace collisions by naming it `SLURM_ML_CACHE` or `MLAGILITY_ML_CACHE` or similar.

Reply:
Changed to `SLURM_ML_CACHE`